src/TiogaBlock.cpp

```diff
@@ -15,9 +16,7 @@
 #include "TiogaMeshInfo.h"
 #include "tioga.h"
 
-extern "C" {
 double computeCellVolume(double xv[8][3],int nvert);
```

Comment:
I think we talked about removing this line since it is actually implemented in tioga?

Reply:
Right. I removed the definition, and added an include statement to include the relevant Tioga header file where computeCellVolume is defined.